spec/clowne/adapters/sequel/associations/one_to_one_spec.rb

```diff
@@ -79,16 +79,16 @@ class HistoryCloner < Clowne::Cloner
       it 'pass params to child cloner' do
         expect(subject.account).to be_new
         expect(subject.account.to_hash).to include(
```

Comment:
Maybe, we can use here `have_attributes` matcher? I.e.:

```ruby
expect(subject.account).to have_attributes(
  ...
)
```

Reply:
Good idea :+1: https://github.com/palkan/clowne/pull/12/commits/e53df3bdc4a15a914d5c71e1276c57b5f3125ff8